src/app/expense/expense.module.ts

```diff
@@ -3,15 +3,21 @@ import { CommonModule } from '@angular/common';
 import { AddEditExpenseComponent } from './add-edit-expense/add-edit-expense.component';
 import { ExpenseListComponent } from './expense-list/expense-list.component';
 
+import { RouterModule,Routes } from '@angular/router';
 
+const expenseRoutes : Routes = [
+  {path:'eList', component :ExpenseListComponent},
+  {path:'eAddEdit', component :AddEditExpenseComponent}
+]
```

Comment:
1. Use better naming convention for all route path, so 'eList' can be 'expense-list' or 'list'
2. Use hyphen between name and avoid camelCase